packages/grant-explorer/src/features/api/rounds.ts

```diff
@@ -3,19 +3,22 @@ import { RoundMetadata } from "./round";
 import { MetadataPointer } from "./types";
 import { fetchFromIPFS } from "./utils";
 import { ethers } from "ethers";
+import { allChains } from "../../app/chainConfig";
 
 interface GetRoundOverviewResult {
   data: {
     rounds: RoundOverview[];
   };
 }
 
+/*TODO: what is this */
```

Comment:
Would be nice to have an explanation here before merging. Who knows about this?

Reply:
@bhargavaparoksham I see you were the one who added it, if I remember correctly this was a temporary measure to fix a production issue, would this be safe to delete now?

Reply:
All these rounds are old and won't display because the donation period has passed, so afaik it's safe to delete them. i added the names of the rounds to the pr.